lib/rpcServer/commands/platform/applyStateTransitionHandlerFactory.js

```diff
@@ -0,0 +1,42 @@
+const Validator = require('../../../utils/Validator');
+const argsSchema = require('./schemas/applyStateTransition');
+
+const validator = new Validator(argsSchema);
+
+/**
+ *
+ * @param {jaysonClient} rpcClient
+ * @param {handleAbciResponse} handleAbciResponse
+ * @returns {applyStateTransitionHandler}
+ */
+function applyStateTransitionHandlerFactory(rpcClient, handleAbciResponse) {
```

Comment:
I think it's better to pass the validator to the factory. It can be useful for unit tests.

Reply:
Fixed